docs/source/whatsnew/pr/incompat-switching-to-perl.rst

```diff
@@ -1 +1,7 @@
+Incompatible change switch to perl
```

Comment:
I know it's unrelated, but I was annoyed to have to fix this on my local machine, with I'm running sphinx with warnings as errors.